lisp/forge-commands.el

```diff
@@ -95,6 +95,8 @@ Takes the pull-request as only argument and must return a directory."
   [["Configure"
     ("a  " "add repository to database" forge-add-repository)
     ("r  " "forge.remote"  forge-forge.remote)
+    ("t l" "max topics to pull" forge-forge.max-topics
+     :if (lambda () (forge-github-repository-p (forge-get-repository nil))))
```

Comment:
Not sure if that is possible, but have you tried moving that into the infix definition?

Reply:
I was just mirroring an example of one further up :smile:

Reply:
Yeah, no need to change that.

But please it last and on "level 7".